app/controllers/people_controller.rb

```diff
@@ -343,6 +343,7 @@ def dependent_same_address_as_primary(dependent)
     dep_address.city = @person_new_home_address.city
     dep_address.state = @person_new_home_address.state
     dep_address.zip = @person_new_home_address.zip
+    dep_address.county = @person_new_home_address.county if @person_new_home_address.county.present?
     dep_address.save!
   end
```

Comment:
Don't we have a problem with the comparison for `Address#same_address` in that it doesn't take into account that a county may have changed?  Won't this fail to do the expected thing if counties are/aren't different for the primary and dependent?

Does this comparison problem also extend to other 'address-like' models in the engines?

We will need to broadcast person update events for each dependent if we choose to change the address - updating a child doesn't always trigger the update notification for the parent document, as we have discovered.

Consider refactoring this into an operation - it will make testing easier.